website/docs/config-file.md

```diff
@@ -98,3 +98,20 @@ size = 18
 [font.features]
 MonoLisa = [ "+ss01", "+ss07", "+ss11", "-calt", "+ss09", "+ss02", "+ss14", "+ss16", "+ss17" ]
 ```
+
+Specify font weight:
+
+```toml
+[font]
+size = 19
+hinting = "full"
+edging = "antialias"
+
+[[font.normal]]
+family = "JetBrainsMono Nerd Font Propo"
+style = "W400"
+
```

Comment:
I think we could add a second element here to demonstrate fallback fonts:
```toml
[[font.normal]]
family = "Noto Sans CJK SC"
style = "Normal"
```

I thought that would be enough, but it may not be actually. Checking the code, it looks like it fallbacks to normal only when a style is not found, and not just for finding fallbacks. So, it's possible that the fallbacks have to be duplicated for all weights. 

That needs some testing to determine if it's the case or not. But I think fallbacks like that is something we do want to support to avoid duplication.

Reply:
I'll trying to modify the fallback logic and add some test.

Reply:
To be clear, do you mean
```toml
[[font.normal]]
family = "not found1"
style = "W400"

[[font.normal]]
family = "menlo"

[[font.bold]]
family = "not found2" #
style = "W600"
```

In this situation, bold font should fallback to menlo instead of builtin fira code(current behaviour).

Reply:
Yes, I think that behaviour makes the most sense, since missing style elements falls back to the normal style, so the fallbacks should work the same way. 

In this case, the full fallback list for bold would be `not found2`, `not found1`, `menlo`, in that order.